code/autoupdater/sha256.c

```diff
@@ -0,0 +1,158 @@
+/*********************************************************************
```

Comment:
It's only used by the autoupdater for now, though. We can move it later if other components start needing it.